augly/image/transforms.py

```diff
@@ -212,6 +212,37 @@ def apply_transform(
         """
         return F.brightness(image, factor=self.factor, metadata=metadata)
 
+class Skew(BaseTransform):
+    def __init__(self, level: float = 5.0, max_level: float = 10.0, axis: str = "x-axis"):
```

Comment:
Can you please use integer indices for axis instead of text? It is best to keep the library similar to `numpy` and `torch` interfaces.

Reply:
Thanks for your comment @hazirbas sir !!

I've made suggested changes into files.